internal/k8s/copy.go

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 Authors of Cilium
```

Comment:
nit: The copyright year should be 2021. This also applies to other files.

Reply:
my bad :(